src/expandableCalendar/index.js

```diff
@@ -101,10 +103,15 @@ class ExpandableCalendar extends Component {
 
     this.state = {
       deltaY: new Animated.Value(startHeight),
-      headerDeltaY: new Animated.Value(this.props.initialPosition === POSITIONS.CLOSED ? 0 : -HEADER_HEIGHT),
-      position: props.initialPosition
+      headerDeltaY: new Animated.Value(props.initialPosition === POSITIONS.CLOSED ? 0 : -HEADER_HEIGHT),
+      position: props.initialPosition,
+      screenReaderEnabled: undefined
     };
 
+    AccessibilityInfo.isScreenReaderEnabled().then((screenReaderEnabled) => {
```

Comment:
I actually wanted to add a listener here since VoiceOver is a toggle but I need to test it on a real device first. Anyways, I changed the state to false in the meantime.

Reply:
While voice over is a mode the user can toggle off and on. 
They never really do. Blind people are alway in VoiceOver mode.
The toggle will be only for us when QAing, so not sure it's necessary

Reply:
Ye, makes sense.